src/proposals/20250306/SparkEthereum_20250306.sol

```diff
@@ -0,0 +1,141 @@
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED,
+            flashloanable:         EngineFlags.DISABLED,
+            ltv:                   65_00,
+            liqThreshold:          70_00,
+            liqBonus:              8_00,
+            reserveFactor:         15_00,
+            supplyCap:             250,
+            borrowCap:             0,
+            debtCeiling:           0,
+            liqProtocolFee:        10_00,
+            eModeCategory:         3
+        });
+
+        listings[1] = IEngine.Listing({
+            asset:       Ethereum.TBTC,
+            assetSymbol: 'tBTC',
+            priceFeed:   AGGOR_BTCUSD_ORACLE,
+            rateStrategyParams:                Rates.RateStrategyParams({
+                optimalUsageRatio:             _bpsToRay(60_00),
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            _bpsToRay(4_00),
+                variableRateSlope2:            _bpsToRay(300_00),
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseStableRateOffset:          0,
+                stableRateExcessOffset:        0,
+                optimalStableToTotalDebtRatio: 0
+            }),
+            enabledToBorrow:       EngineFlags.ENABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED,
+            flashloanable:         EngineFlags.ENABLED,
+            ltv:                   65_00,
+            liqThreshold:          70_00,
+            liqBonus:              8_00,
+            reserveFactor:         20_00,
+            supplyCap:             125,
+            borrowCap:             25,
+            debtCeiling:           0,
+            liqProtocolFee:        10_00,
+            eModeCategory:         0
+        });
+
+        return listings;
+    }
+
+    function collateralsUpdates()
+        public pure override returns (IEngine.CollateralUpdate[] memory collateralUpdates)
+    {
+        collateralUpdates = new IEngine.CollateralUpdate[](1);
+
+        collateralUpdates[0] = IEngine.CollateralUpdate({
+            asset:          Ethereum.CBBTC,
+            ltv:            EngineFlags.KEEP_CURRENT,
+            liqThreshold:   EngineFlags.KEEP_CURRENT,
+            liqBonus:       EngineFlags.KEEP_CURRENT,
+            debtCeiling:    EngineFlags.KEEP_CURRENT,
+            liqProtocolFee: EngineFlags.KEEP_CURRENT,
+            eModeCategory:  3
+        });
+    }
+
+    function _postExecute() internal override {
+        // Seed the new LBTC pool
+        IERC20(Ethereum.LBTC).approve(address(LISTING_ENGINE.POOL()), 0.0001e8);
+        LISTING_ENGINE.POOL().supply(Ethereum.LBTC, 0.0001e8, address(this), 0);
```

Comment:
We used to deposit/supply `1e18`, but now it's `0.0001e8` and `0.0001e18`. 
Do you think the amount doesn't make a difference, so can send less?

Reply:
Its BTC denominated, so roughly $8